### DISCUSSION TITLE
vc_version in the release package

### DISCUSSION THREAD

**Discussion by TheCleric** [educelab/volume-cartographer#79] [Q&A] [answered]
When filling out a bug report, the version of volume-cartographer you are using is requested by running `vc_version`. I see this in the source code, but it doesn't look like it's in the actual releases (both on GitHub and downstream on Homebrew). Rather than creating a bug for this, I want to make sure I'm not missing something terribly obvious. Is vc_version available on releases (and I'm just blind)?

**Comment by csparker247** [comment]
It is in the utilities install list: https://github.com/educelab/volume-cartographer/blob/4589ba21e949ae272ba91149bc25f8b834a8d1a2/utils/CMakeLists.txt#L176

But is not in the Homebrew Cask install list: https://github.com/educelab/homebrew-casks/blob/main/Casks/volume-cartographer.rb#L39 It should be added here.

It should be in the Docker images, though.

**Reply by TheCleric** [reply]
I thought about updating that to add `binary "bin/vc_version"` on (currently) line 40 of the homebrew cask, but referencing the zip file that homebrew would be using (such as `https://github.com/educelab/volume-cartographer/releases/download/v2.26.0-rc.3/VC-2.26.0-rc.3-Darwin-arm64.zip`) I don't see that executable in the bin directory (or elsewhere in that package unless I'm missing something).

**Reply by csparker247** [reply]
That's weird. It's there when I download that zip:
```
% unzip -vl VC-2.26.0-rc.3-Darwin-arm64.zip | grep vc_version
   54816  Defl:N     3955  93% 12-20-2023 21:30 8fa67612  bin/vc_version
```

**Comment by csparker247** [comment, marked as answer]
I just added `vc_version` to the cask install list and it installs on my machine:

```shell
% brew reinstall --no-quarantine educelab/casks/volume-cartographer
% vc_version
volume-cartographer 2.26.0 (be77dc0533f6d6c2efbf691d0d2538ca86f04d3d)
```

Let me know if this doesn't work for you!

**Reply by TheCleric** [reply]
I feel like such a fool. This does indeed work now. And going back to my extracted zip I see the vc_version binary. I don't know what I was thinking yesterday. Not enough coffee I suppose.

**Reply by csparker247** [reply]
Haha, no worries! We all need more coffee. Glad it works and thanks for reporting.
